export const signIn = async () => {
    try {
        
        
    } catch (error) {
      console.error("Error fetching data:", error);
      throw error; 
    }
  };